xls/dslx/type_system/scoped_fn_stack_entry.cc

```diff
@@ -42,10 +37,6 @@ ScopedFnStackEntry::ScopedFnStackEntry(Function& fn, DeduceCtx* ctx,
   ctx->AddFnStackEntry(FnStackEntry::Make(fn, ParametricEnv(), within_proc));
 }
 
-// Called when we close out a scope. We can't use this object as a scope
```

Comment:
q: any reasons those get removed?

Reply:
Ditto. Already present on the .h so this was redundant and in the wrong spot.